examples/ble_nus/src/main.c

```diff
@@ -0,0 +1,222 @@
+/*
+ * Copyright 2023 u-blox
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+ *
+ * A demo on how to set up a NUS (Nordic Uart Service) server or client
+ * using ubxlib.
+ *
+ * IMPORTANT! ubxlib version 1.2 or later is required.
+ *
+ */
+
+#include <stdio.h>
+#include <string.h>
+
+#include "ubxlib.h"
+
+#include "leds.h"
+
+#define SERVER_NAME "NUS-Demo-Server"
+#define SOLID_LED  RED_LED
+#define BLINK_LED  BLUE_LED
+
+// This application can run as either server (peripheral) or client (central).
+// Choose here which to run.
+bool gActAsServer = true;
```

Comment:
Very minor point: some variables non static, is there a reason for that?

Reply:
Laziness I assume :)